tools/Experiments/src/ViewModels/QuietBackgroundProcessesViewModel.cs

```diff
@@ -0,0 +1,218 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using System.Linq;
+using System.Security.Principal;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using DevHome.Common.Helpers;
+using Microsoft.UI.Xaml;
+using Windows.UI.Xaml;
+
+namespace DevHome.Experiments.ViewModels;
+public class QuietBackgroundProcessesViewModel : INotifyPropertyChanged
+{
+    private readonly TimeSpan _zero;
+    private readonly bool _isElevated;
+    private readonly bool _validOsVersion;
+
+    public QuietBackgroundProcessesViewModel()
+    {
+        _zero = new TimeSpan(0, 0, 0);
+
+        var osVersion = Environment.OSVersion;
+        _validOsVersion = osVersion.Version.Build >= 26024;
+        _validOsVersion = true;
+
+        if (!_validOsVersion)
+        {
+            TimeLeft = "This feature requires OS version 10.0.26024.0+";
+            return;
+        }
+
+        using (WindowsIdentity identity = WindowsIdentity.GetCurrent())
+        {
+            WindowsPrincipal principal = new WindowsPrincipal(identity);
+            _isElevated = principal.IsInRole(WindowsBuiltInRole.Administrator);
+        }
+
+        // Resume countdown if there's an existing quiet window
+        if (GetIsActive())
+        {
+            _isToggleOn = true;
+            var timeLeftInSeconds = GetTimeRemaining();
+            StartCountdownTimer(timeLeftInSeconds);

```

Comment:
is `StartCountdownTimer` synchronous?  Won't this halt execution until the timer is done?